makedebian.pl

```diff
@@ -309,10 +315,16 @@
 rm -f /var/lock/subsys/$baseproduct
 
 if [ "$inetd" != "1" ]; then
-	/etc/$baseproduct/stop >/dev/null 2>&1 </dev/null
-	/etc/$baseproduct/start >/dev/null 2>&1 </dev/null
-	if [ "\$?" != "0" ]; then
-		echo "W: $ucproduct server cannot be restarted. It is advised to restart it manually by\nrunning \\"/etc/webmin/restart-by-force-kill\\" when upgrade process is finished"
+	if [ "\$justinstalled" = "1" ]; then
+		/etc/$baseproduct/start >/dev/null 2>&1 </dev/null
+		if [ "\$?" != "0" ]; then
+			echo "E: Webmin server cannot be started. It is advised to start it manually\n   by running \\"/etc/webmin/restart-by-force-kill\\" command"
+		fi
+	else
+		/etc/$baseproduct/.post-install >/dev/null 2>&1 </dev/null
```

Comment:
I agree with the idea of sending a HUP signal here instead of doing stop/start , but how does .post-install send a HUP signal in all cases? Don't forget that this has to work even for systems without systemd

Reply:
I know and I didn't forget. I have answered your question in the comment above, here - https://github.com/webmin/webmin/pull/1677#discussion_r912508947

Reply:
Ok I see now .. thanks!